packages/zoom/api.js

```diff
@@ -8,7 +7,7 @@ class Api extends OAuth2Requester {
         this.baseUrl = `https://api.zoom.us/v2/`;
         this.client_id = process.env.ZOOM_CLIENT_ID;
         this.client_secret = process.env.ZOOM_CLIENT_SECRET;
-        this.redirect_uri = 'https://29a3f7035dbb.ngrok.io';
+        this.redirect_uri = process.env.REDIRECT_URI;
```

Comment:
You can remove this line. The parent class will handle the `redirect_uri` attribution. The user needs to pass the variable to work properly, but it is best than assigning the env to the redirect every time